c++/include/orc/Reader.hh

```diff
@@ -32,13 +32,34 @@
 #include <set>
 #include <string>
 #include <vector>
+#include <atomic>
 
 namespace orc {
 
   // classes that hold data members so we can maintain binary compatibility
   struct ReaderOptionsPrivate;
   struct RowReaderOptionsPrivate;
 
+  /**
+   * Expose the reader metrics including the latency and
+   * number of calls of the decompression/decoding/IO modules.
```

Comment:
Please document the relation of the latency counters. IIUC, ReaderInclusiveLatencyUs contains the time of decoding latency. Decoding latency contains the time of decompression latency which finally contains the time of IO blocking.

Reply:
Done. I added some comments to the metrics latency fields.